Dockerfile

```diff
@@ -48,6 +48,8 @@ ENV PATH="/opt/venv/bin:${PATH}"
 RUN python3 -m pip install --no-cache-dir -U pip==22.0.2 setuptools==60.6.0 wheel==0.37.1
 COPY cvat/requirements/ /tmp/requirements/
 RUN DATUMARO_HEADLESS=1 python3 -m pip install --no-cache-dir -r /tmp/requirements/${DJANGO_CONFIGURATION}.txt
+COPY utils/dataset_manifest/ /tmp/dataset_manifest/
+RUN python3 -m pip install --no-cache-dir -r /tmp/dataset_manifest/requirements.txt
```

Comment:
It would be useful to merge this `pip install` with the previous one, so that if there are any conflicts between the two requirements files, pip would complain about them.